### PR TITLE
removed outlines, excessive comments, and fixed font color in form

### PR DESCRIPTION
**Pull request by dylanh4**
Made a number of direct commits to master, which was a mistake. Made additional necessary updates on this branch.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -40,7 +40,6 @@ var makeHeaderRow = function() {
   var thEl = document.createElement('th');
   thEl.textContent = ' ';
   trEl.appendChild(thEl);
-  // cookieTable.appendChild(trEl);
```
Comment (nhuntwalker) [inline app.js:43]: Can't comment at the top right now, but you're missing 'use strict'; at the top.

Comment (dylanh4) [inline app.js:43]: Thanks @nhuntwalker  - just made the update!

Comment (dylanh4) [inline app.js:43]: The change is on master though so totally understand if my pr gets marked down for the omission.

File: app.js [hunk 2]
```diff
@@ -91,14 +89,16 @@ var seaTac = new PatCookiesLocation('SeaTac', 3, 24, 1.2, 'seaTacID');
 var seaCenter = new PatCookiesLocation('Seattle Center', 11, 38, 3.7, 'seaCenterID');
 var capHill = new PatCookiesLocation('Capitol Hill', 20, 38, 2.3, 'capHillID');
 var alki = new PatCookiesLocation('Alki', 2, 16, 4.6, 'alkiID');
+
 //appended to table
 wholeTable();
+
 //grabbing form
```
Comment (nhuntwalker) [inline app.js:96]: Comment unnecessary

File: app.js [hunk 3]
```diff
@@ -91,14 +89,16 @@ var seaTac = new PatCookiesLocation('SeaTac', 3, 24, 1.2, 'seaTacID');
...
 var cookieForm = document.getElementById('cookie_form');
+
 //this is our cookie handler
```
Comment (nhuntwalker) [inline app.js:99]: Comment unnecessary

File: app.js [hunk 4]
```diff
@@ -107,9 +107,10 @@ function handleCookieForm(event) {
   var phoneNum = document.getElementById('phone_num');
 //adding the newCookieStand to the cookieStandsArray
```
Comment (nhuntwalker) [inline app.js:108]: Comment unnecessary

File: app.js [hunk 5]
```diff
@@ -107,9 +107,10 @@ function handleCookieForm(event) {
...
   var newCookieStand = new PatCookiesLocation(locationName.value, minCust.value, maxCust.value, avgCookies.value, address.value, phoneNum.value);
-  // console.log(cookieStandsArray);
+
 //adding the new table row
```
Comment (nhuntwalker) [inline app.js:111]: Comment unnecessary

**Pull request merged**